templates/telemetry/fxa_all_events_v1/view.sql

```diff
@@ -45,10 +45,26 @@ WITH
       jsonPayload.fields.event_type
     FROM
       `moz-fx-data-derived-datasets.telemetry.fxa_content_events_v1`
+  ),
+
+  fxa_oauth_events AS (
+    SELECT
+      timestamp AS submission_timestamp,
+      jsonPayload.fields.user_id,
+      CAST(NULL AS STRING) AS country,
```

Comment:
were these missing from the oauth events?

Reply:
Correct. I do not see any of these fields present in the schema for oauth events.